ironfish-cli/src/commands/accounts/balance.ts

```diff
@@ -82,6 +84,8 @@ export class BalanceCommand extends IronfishCommand {
     const unconfirmedDelta = unconfirmed - confirmed
 
     this.log(`Account: ${response.account}`)
+    this.log(`Head Hash: ${response.blockHash || 'NULL'}`)
+    this.log(`Head Sequence: ${response.sequence || 'NULL'}`)
```

Comment:
Can this be in human format?

`Your balance is calculated on the chain at a block ${hash} at sequence {sequence}` or something like it?